### PR TITLE
Fix merge / test regressions

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

File: tests/control/DirectorTest.php [hunk 1]
```diff
@@ -402,6 +402,9 @@ public function testUnmatchedRequestReturns404() {
 	}
 
 	public function testIsHttps() {
+		if(!TRUSTED_PROXY) {
```
Comment (tractorcow) [inline tests/control/DirectorTest.php]: See https://github.com/silverstripe-labs/silverstripe-travis-support/commit/09db98db8d141eb5ee23da2a6c4d124461e675b5

File: thirdparty/simpletest/url.php [hunk 2]
```diff
@@ -419,13 +419,16 @@ function asString() {
             $scheme .= "://";
             $host = $this->getHost();
         }
+		if ($this->getPort() && $this->getPort() != 80 ) {
```
Comment (tractorcow) [inline thirdparty/simpletest/url.php]: This was backported from the newer version of SimpleTest module.

File: tests/control/DirectorTest.php (hunk 1 above)
Comment (hafriedlander) [inline tests/control/DirectorTest.php]: Seems like the TestSkipped string is backwards? !TRUSTED_PROXY means untrusted proxy, but test says test is skipped if !untrusted proxy.

Comment (tractorcow) [inline tests/control/DirectorTest.php]: Fixed

**Pull request merged**